### PR TITLE
Add qqp readme

### PR DESCRIPTION
**Pull request by CPFLAME**
这个pr 要做的:

- [x] 添加qqp的readme
- [x] 添加qqp的数据下载脚本

### CODE REVIEW

File: projects/QQP/dataset/download_qqp_data.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import sys
+
+sys.path.append(".")
+from libai.utils.file_utils import get_data_from_cache  # noqa
+
+VOCAB_URL = "https://oneflow-static.oss-cn-beijing.aliyuncs.com/ci-files/dataset/libai/bert_dataset/bert-base-chinese-vocab.txt"  # noqa
+QQP_TRAIN_URL = "https://oneflow-static.oss-cn-beijing.aliyuncs.com/ci-files/dataset/libai/QQP/train.tsv"  # noqa
+QQP_TEST_URL = (
+    "https://oneflow-static.oss-cn-beijing.aliyuncs.com/ci-files/dataset/libai/QQP/dev.tsv"  # noqa
```
Comment (CPFLAME) [inline projects/QQP/dataset/download_qqp_data.py:24]: 这里 python black 会自动修正给我加个括号并换行..

File: projects/QQP/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+```
+projects/QQP/QQP_DATA/
+│---bert-base-chinese-vocab.txt
+│---train.tsv    
+│---dev.tsv
+```
+
+## Start Training
+Run command:
+```shell
```
Comment (rentainhe) [inline projects/QQP/README.md:24]: 这儿是不是加一下
`cd libai`或者`cd /path/to/libai`会清晰一些

File: projects/QQP/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+bash tools/train.sh tools/train_net.py projects/QQP/configs/config_qqp.py $num_gpus
+```
+where `$num_gpus` indicates the number of GPUs. If you want to run the distributed program, you can change it, for example:
+```bash
+bash tools/train.sh tools/train_net.py projects/text_classification/configs/config.py 4
+```
+
+Before running the program, you should modify the `projects/QQP/configs/config_qqp.py` file. Modification fields include but are not limited to task_name, data_dir, vocab_file, model hyperparameter, learning rate, and batch size, and so on.
```
Comment (rentainhe) [inline projects/QQP/README.md:32]: modify the `projects/QQP/configs/config_qqp.py` depend on your own needs 感觉会好点

File: projects/QQP/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+│---train.tsv    
+│---dev.tsv
+```
+
+## Start Training
+Run command:
+```shell
+bash tools/train.sh tools/train_net.py projects/QQP/configs/config_qqp.py $num_gpus
+```
+where `$num_gpus` indicates the number of GPUs. If you want to run the distributed program, you can change it, for example:
```
Comment (Ldpe2G) [inline projects/QQP/README.md:27]: For example, if you want to run the distributed program on 4 GPUs, you can set to 4. 

这样好些

File: projects/QQP/README.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+We provide code for finetuning on qqp dataset based on pretrained bert model. Differently, the language model is fine-tuned and pretrained from **Chinese** vocabulary. 
+
+## Download Data
+
+Run command:
+```bash
+python3 projects/QQP/dataset/download_qqp_data.py
+```
+Data and Vocabulary will be downloaded in `projects/QQP/QQP_DATA/`
+> NOTE: pretrained model will not be provided due to privacy reasons
```
Comment (L1aoXingyu) [inline projects/QQP/README.md:14]: 可以让他们去下载 megatron 的 pretrain weight

**Review by Ldpe2G — APPROVED** [review]

File: projects/QQP/README.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+```shell
+cd /path/to/libai_root
+bash tools/train.sh tools/train_net.py projects/QQP/configs/config_qqp.py $num_gpus
+```
+where `$num_gpus` indicates the number of GPUs. For example, if you want to run the distributed program on 4 GPUs, you can set to 4.:
+```bash
+bash tools/train.sh tools/train_net.py projects/text_classification/configs/config.py 4
+```
+
+Before running the program, you should modify the `projects/QQP/configs/config_qqp.py` depend on your own needs . Modification fields include but are not limited to task_name, data_dir, vocab_file, model hyperparameter, learning rate, and batch size, and so on.
```
Comment (kaijieshi7) [inline projects/QQP/README.md:34]: 第一句话最后的句号之前多了一个空格

Comment (CPFLAME) [reply projects/QQP/README.md:34]: 已改
